docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_use_custom_expectations.md

```diff
@@ -5,6 +5,9 @@ import Prerequisites from '../creating_custom_expectations/components/prerequisi
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+import CLIRemoval from '/docs/components/warnings/_cli_removal.md'
+
+<CLIRemoval />;
```

Comment:
```suggestion
<CLIRemoval />
```
I checked, and the `;` is being rendered on the page itself.  These are HTML tags rather than code, so the `;` is superfluous.  You should be able to do a find/replace for all of those.

Reply:
@Rachel-Reverie - done. Thanks!